src/main/java/radiounit/ConcreteRadioUnit.java

```diff
@@ -26,9 +26,29 @@ public ConcreteRadioUnit(String ipAddress, String name, Vendor vendor, RatType r
 		this.ratType = ratType;
 		this.alarmStatus = AlarmStatusLevel.NO_ALARM;
 		this.state = new IdleState(this);
-		// TODO: instantiate a RadioCommandExecutor
-		// TODO: get RadioCommandExecutorFactory instance here.
-
+		this.postActivationComplete = false;
+		this.signalScalingComplete = false;
+
+		switch (this.vendor) {
+		case ERICSSON:
+			switch (this.ratType) {
+			case LTE:
+				cmdExecutor = EricssonLteCommandExecutorFactory.getInstance().createRadioCommandExecutor();
+				break;
+			case WCDMA:
+//				cmdExecutor = EricssonWcdmaCommandExecutorFactory.getInstance().createRadioCommandExecutor();
```

Comment:
For these three, commented out lines, we're just waiting to merge the three executor/receiver classes in the other PRs